### ISSUE TITLE
Minor capitalization mistake in the core title

### ISSUE THREAD

**Issue by fr500** [libretro/libretro-meta#17]
_From @GeneralFailer on April 11, 2015 9:14_

The emulator name in the core metadata is spelled as SNES9x even though it's spelled as Snes9x almost everywhere in the official resources of the original emulator. It is sometimes spelled as Snes9X there and in the UI, which is an issue I've made [an entry](https://github.com/snes9xgit/snes9x/issues/69) in the emulator's issue tracker about, which is a different story. The point is, this spelling is pretty much incorrect and can potentially cause some, albeit small, confusion. 
This issue is relevant to Snes9x Next too, though it's probably more excusable for a fork.

_Copied from original issue: libretro/snes9x#11_

**Comment by fr500** [comment]
_From @Alcaro on April 11, 2015 9:17_

Twinaphex seems to like making up his own capitalizations for various stuff. I've caught him saying bSNES a few times too.

**Comment by fr500** [comment]
_From @Alcaro on April 11, 2015 9:22_

Not sure which of Snes9x and Snes9X is the official one, but they've both existed for quite a while, so I'd say both are correct enough. I'll fix it again if your issue decides on Snes9x.

**Comment by fr500** [comment]
_From @GeneralFailer on April 11, 2015 9:25_

I think that's fine for something like commit descriptions, but not for this kind of stuff. Don't forget to change the Authors field.
The license uses Snes9x spelling if that's of any importance.

**Comment by fr500** [comment]
_From @Alcaro on April 11, 2015 9:40_

Do you mean [this](http://git.io/vvfGe) author field, or is there another one?

Lowercase does indeed seem more common (Wikipedia and the copyright headers both do that), so I'll admit I picked the wrong one. But they've both existed for quite a while, so it's low priority unless the Snes9x devs post a strong opinion in either direction.

**Comment by fr500** [comment]
_From @GeneralFailer on April 11, 2015 9:41_

Yeah, I think that's the one.

**Comment by fr500** [comment]
_From @GeneralFailer on April 11, 2015 11:16_

You know, you guys are a bit sloppy about naming overall. Here's what I've found, might've missed something in the nightly cores:
bNES -> bnes, bSNES -> bsnes, DeSmuME svn -> DeSmuME (SVN), dosbox -> DOSBox, gambatte -> Gambatte, handy -> Handy, Mednafen Neopop -> Mednafen NeoPop, Mupen64plus -> Mupen64Plus, prboom -> PrBoom
EDIT: By the way, you seem to have forgot to fix snes9x_libretro.info in Snes9x Next.

**Comment by fr500** [comment]
_From @Alcaro on April 11, 2015 12:30_

As I said, Twinaphex likes making up his own capitalizations for various stuff. Fixed a bunch of bSNES typos, I'll leave the non-SNES cores to someone else.

There is no .info in the snes9x-next repo, and the one in libretro-super is correct. The nightlies update only once per day, and the fix is three hours old; if it's still there tomorrow, give me the link so I can find which one you're thinking of.

I'd recommend opening this issue on libretro-super instead. That is the correct location for multi-core bugs; here in the Snes9x repo, the relevant people are likely to miss it.

**Comment by fr500** [comment]
_From @GeneralFailer on May 15, 2015 19:26_

@Alcaro
I guess there pretty much is a conclusion now.
